.github/CODEOWNERS

```diff
@@ -101,9 +101,9 @@
 /packages/plugins                               @gziolo @adamsilverstein
 
 # Rich Text
-/packages/format-library                        @ellatrix @daniloercoli @sergioestevao @etoledom
-/packages/rich-text                             @ellatrix @daniloercoli @sergioestevao @etoledom
-/packages/block-editor/src/components/rich-text @ellatrix @daniloercoli @sergioestevao @etoledom
+/packages/format-library                        @ellatrix @etoledom @guarani @cameronvoell
```

Comment:
@SergioEstevao , can you actually put Cameron as the third contact in the list, so we can have better platform coverage (Eduardo is primarily iOS and Cameron is primarily Android) when people read through that list?